eodhistoricaldata/src/endpoint/price.ts

```diff
@@ -0,0 +1,45 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
```

Comment:
```suggestion
import { ExecuteWithConfig, Config } from '@chainlink/types'
```
A recent PR made ExecuteWithConfig a generic. (This applies across all EAs in this PR)